tests/ccgbugs/tcodegen_anonymous_aggregate.nim

```diff
@@ -0,0 +1,34 @@
+discard """
+targets: "c cpp"
```

Comment:
This might be handled by defaultCategory in testament.Nim, have a look and it might default to these targets.